src/sentry/static/sentry/app/stores/organizationStore.jsx

```diff
@@ -0,0 +1,30 @@
+import Reflux from 'reflux';
+
+import OrganizationActions from 'app/actions/organizationActions';
+import OrganizationsActions from 'app/actions/organizationsActions';
+
+const OrganizationStore = Reflux.createStore({
+  listenables: OrganizationActions,
+
+  init() {
+    this.reset();
+
+    // also listen to updates from OrganizationsActions triggered from settings
+    this.listenTo(OrganizationsActions.update, this.onUpdate);
+  },
+
+  reset() {
+    this.org = {};
```

Comment:
Should this be `null`?

Reply:
Hmm not quite sure, the other stores have either [] or {} as the initial state after a reset. Would it make more sense to start off as null for this store though?

Reply:
Maybe, otherwise components using this store may think an organization is loaded and available to use

Reply:
sounds good